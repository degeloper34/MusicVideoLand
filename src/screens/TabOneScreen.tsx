import {StyleSheet, Text, View} from "react-native";
import Colors from "../constants/Colors";
import Layout from "../constants/Layout";

export default function TabOneScreen() {
  return (
    <View style={styles.container}>
      <Text style={styles.title}>Tab One</Text>
      <View style={styles.separator} />
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    alignItems: "center",
    justifyContent: "center",
    backgroundColor: Colors.black,
  },
  title: {
    fontSize: Layout.l,
    fontWeight: "bold",
    color: Colors.white,
  },
  separator: {
    marginVertical: Layout.xxl,
    height: 1,
    width: "80%",
    backgroundColor: Colors.white,
  },
});
