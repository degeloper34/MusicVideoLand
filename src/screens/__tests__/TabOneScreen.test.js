import renderer from "react-test-renderer";
import React from "react";
import TabOneScreen from "../TabOneScreen";

describe("TabOneScreen Tests", () => {
  it("TabOneScreen | Render without issues", () => {
    const screen = renderer.create(<TabOneScreen />).toJSON();

    expect(screen).toMatchSnapshot();
  });
});
