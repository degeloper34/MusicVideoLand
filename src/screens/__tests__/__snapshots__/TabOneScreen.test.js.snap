// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabOneScreen Tests TabOneScreen | Render without issues 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#000",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#fff",
        "fontSize": 20,
        "fontWeight": "bold",
      }
    }
  >
    Tab One
  </Text>
  <View
    style={
      Object {
        "backgroundColor": "#fff",
        "height": 1,
        "marginVertical": 30,
        "width": "80%",
      }
    }
  />
</View>
`;
